field.cpp

```diff
@@ -709,6 +708,11 @@ int32 field::get_spsummonable_count_fromex_rule4(card* pcard, uint8 playerid, ui
 		if(!is_location_useable(playerid, LOCATION_MZONE, 6))
 			flag |= 1u << 6;
 	}
+	uint32 linked_zone = 0;
```

Comment:
even if duel_rule is 4, it include non-linked extra monster zones
```suggestion
	uint32 allowed_zone = 0;
```

Reply:
what about `rule_zone`